NEWS.md

```diff
@@ -2,7 +2,16 @@ MXNet Change Log
 ================
 ## 1.2.1
 ### Deprecations
-- An incorrect [usage](https://github.com/apache/incubator-mxnet/issues/11091) of `save_params` was advertised in the gluon book which led to MXNet users depending on the incorrect usage and developing a hack around it. A change was made to the internal structure of the `.params` file saved by `save_params` to resolve a bug. This led to user scripts with the above mentioned hack to break. To fix this, `save_params` and `load_params` APIs have been reverted to previous format and marked as deprecated. New APIs: `save_parameters` and `load_parameters` have been added for the new format. All scripts to save and load parameters for a Gluon model should now use the new API for `save_parameters` and `load_parameters`. If your model is hybridizable and you want to export a serialized structure of the model as well as parameters you need to use the `export` API and the newly added `imports` API instead of `save_params` and `load_params` API. For more details, Please see: [issue](https://github.com/apache/incubator-mxnet/issues/11091), [PR](https://github.com/apache/incubator-mxnet/pull/11127).
+An incorrect [usage](https://github.com/apache/incubator-mxnet/issues/11091) of `save_params` was advertised in the [gluon book](https://github.com/zackchase/mxnet-the-straight-dope/blob/master/chapter07_distributed-learning/hybridize.ipynb) which led MXNet users to depend on the incorrect usage and developing a hack around it. A change was made to the internal structure of the `.params` file saved by `save_params` to resolve a bug. This led to user scripts with the above mentioned hack to break. To fix this, `save_params` and `load_params` APIs have been reverted to previous format and marked as deprecated. New APIs: `save_parameters` and `load_parameters` have been added for the new format. All scripts to save and load parameters for a Gluon model should now use the new API for `save_parameters` and `load_parameters`. If your model is hybridizable and you want to export a serialized structure of the model as well as parameters you need to use the `export` API and the newly added `imports` API instead of `save_params` and `load_params` API.
```

Comment:
`<new line>`
To allow for backward compatibility and to fix this breaking change, `save_params` and `load_params` APIs have been reverted to their previous behavior as specified in v1.1.0. With v1.2.1, usage of these two APIs will resume their former functionality and a deprecation warning will appear. The functionality introduced to these APIs in v1.2.0 is now renamed to `save_parameters` and `load_parameters` respectively.
`<new line>`
All scripts to save and load parameters for a Gluon model should use the new API for `save_parameters` and `load_parameters`. If your model is hybridizable and you want to export a serialized structure of the model as well as parameters you should migrate your code to use `export` API and the newly added `imports` API instead of `save_params` and `load_params` API.

Note: At this point I'm left wondering what the difference is between `export` and `save_parameters`. Same for `import` and `load_parameters`... Both of these are new and "recommended", but I don't understand what the difference is.

Reply:
should I Link this doc: https://mxnet.incubator.apache.org/tutorials/gluon/save_load_params.html to better explain the difference ?